### DISCUSSION TITLE
How to add new language for translation?

### DISCUSSION THREAD

**Discussion by imhemayatsangin** [serenity-is/Serenity#6238] [Help] [answered]
I just added few languages for the translation purpose, it appears in the language dropdown but whenever i select the language it doesn't translate and it shows the english language only, So can anybody guide me how to add new language to serenity .net core ?

**Comment by asilmertkan** [comment, marked as answer]
1. Open Translation page
2. Select Source Language Dropdown as English
3. Select your added language for target language dropdown 
4. Write your translations and save

**Reply by imhemayatsangin** [reply]
Thanks for the response bro, as I have mentioned it I added the language and also I added translations for that particular language but when I select it from the language bar in the interface it doesn't change that language and still it shows the English language and same English page.

**Reply by VictorTomaili** [reply]
check this https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Web/Initialization/UserCultureProvider.cs

**Reply by imhemayatsangin** [reply]
> check this https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Web/Initialization/UserCultureProvider.cs

Thanks @VictorTomaili .Solved it with your comment.
